common/app/model/dotcomrendering/DotcomRenderingUtils.scala

```diff
@@ -506,7 +506,9 @@ object DotcomRenderingUtils {
       sectionLabel = Localisation(article.content.sectionLabelName)(request), // String
       sectionUrl = article.content.sectionLabelLink, // String
       sectionName = article.metadata.section.map(_.value), // Option[String]
-      subMetaSectionLinks = article.content.submetaLinks.sectionLabels.map(SubMetaLink.apply), // List[SubMetaLink]
+      subMetaSectionLinks = article.content.submetaLinks.sectionLabels
+        .map(SubMetaLink.apply)
+        .filter(_.title.trim.nonEmpty), // List[SubMetaLink]
```

Comment:
The slightly more involved fix would be to make `sectionLabelLink`/`sectionLabelName` optional here: https://github.com/guardian/frontend/blob/b7e48d3751864268ad4b9003a272c43314ecfad6/common/app/model/meta.scala#L669

I would guess that empty string was never the right choice to convey a lack of section link here: https://github.com/guardian/frontend/blob/e2ba6f0c2bf07789eedb0373a86a7e206fc30652/common/app/model/content.scala#L216

Reply:
Totally. Happy to do that :)